sky/cli.py

```diff
@@ -895,32 +896,52 @@ def _make_dag_from_entrypoint_with_overrides(
     return dag
 
 
-def _start_cluster(cluster_name: str,
-                   idle_minutes_to_autostop: Optional[int] = None,
-                   retry_until_up: bool = False):
-    handle = global_user_state.get_handle_from_cluster_name(cluster_name)
-    backend = backend_utils.get_backend_from_handle(handle)
-    assert isinstance(backend, backends.CloudVmRayBackend)
-    with sky.Dag():
-        dummy_task = sky.Task().set_resources(handle.launched_resources)
-        dummy_task.num_nodes = handle.launched_nodes
-    handle = backend.provision(dummy_task,
-                               to_provision=handle.launched_resources,
-                               dryrun=False,
-                               stream_logs=True,
-                               cluster_name=cluster_name,
-                               retry_until_up=retry_until_up)
-    if idle_minutes_to_autostop is not None:
-        backend.set_autostop(handle, idle_minutes_to_autostop)
-    return handle
-
-
-class _NaturalOrderGroup(click.Group):
-    """Lists commands in the order they are defined in this script.
+class _NaturalOrderWithAliasesGroup(click.Group):
```

Comment:
I feel this may be a bit too complicated.

How about we simply don't show `spot status` in `sky -h`, but still (1) make it available and alias the new `spot queue` (2) print the warning?

Example: https://github.com/ray-project/ray/blob/11e408908e1a1cac12f0699c5f970fcabff84801/python/ray/scripts/scripts.py#L2551